### ISSUE TITLE
[OTHER] [Good First Issue Finder app] Fill out Playwright skeleton test: left-side menu name

### ISSUE THREAD

**Issue by bugITwhisperer** [EddieHubCommunity/good-first-issue-finder#377]
### What would you like to share?

Playwright skeleton test in the [Good First Issue Finder application](https://github.com/EddieHubCommunity/good-first-issue-finder) requires filling out - as per information [here](https://finder.eddiehub.io/docs/testing)

```
test.fixme(
  'is the left-side menu containing an element with the same name as the page headline',
  async ({ page }) => {
    // navigate to the docs landing page /docs
    // check if the Introduction element in the left menu is holding the active class
  },
);
```

The introduction.spec.ts [file](https://github.com/EddieHubCommunity/good-first-issue-finder/blob/main/tests/docs/introduction.spec.ts) has to be updated.

### Additional information
Once the issue is approved, please assign it to me.

**Comment by bugITwhisperer** [comment]
This one could will be closed as a duplicate of https://github.com/EddieHubCommunity/good-first-issue-finder/issues/377
One issue for adding test cases should cover it all
